js-miniapp-sample/src/pages/app-close-alert.js

```diff
@@ -0,0 +1,242 @@
+import {
+  Button,
+  Container,
+  TextField,
+  Grid,
+  makeStyles,
+  Typography,
+} from '@material-ui/core';
+
+import Checkbox from '@mui/material/Checkbox';
+import FormControlLabel from '@mui/material/FormControlLabel';
+import { red } from '@material-ui/core/colors';
+import MiniApp, { CloseAlertInfo } from 'js-miniapp-sdk';
+
+const useStyles = makeStyles((theme) => ({
+  root: {
+    height: '90%',
+    width: '100%',
+  },
+  grid: {
+    display: 'flex',
+    height: '20%',
+    flexDirection: 'column',
+    justifyContent: 'center',
+    padding: '20px',
+  },
+  table: {
+    minWidth: '80%',
+  },
+  content: {
+    height: '50%',
+    justifyContent: 'space-between',
+    display: 'flex',
+    flexDirection: 'column',
+    alignItems: 'center',
+    fontSize: 18,
+    color: theme.color.primary,
+    fontWeight: 'bold',
+  },
+  contentSection: {
+    height: '30%',
+    padding: '10px',
+  },
+  label: {
+    display: 'block',
+    fontSize: 12,
+    width: '100%',
+    color: theme.color.primary,
+  },
+  imageBox: {
+    height: '250px',
+    margin: '20px',
+  },
+  imageBoxContent: {
+    height: '250px',
+    objectFit: 'contain',
+  },
+  error: {
+    color: red[500],
+    marginTop: 10,
+  },
+  red: {
+    color: red[500],
+  },
+}));
+
+export const initialState = {
+  isLoading: false,
+  isError: false,
+  error: null,
+  isSuccess: false,
+  inputError: null,
+};
+
+type State = {
+  isLoading: ?boolean,
+  isError: ?boolean,
+};
+
+type Action = {
+  type: string,
+  miniAppError: MiniAppError,
+  inputError: ?string,
+};
+
+export const dataFetchReducer = (state: State, action: Action) => {
+  switch (action.type) {
+    case 'FETCH_INIT':
+      return {
+        ...state,
+        isLoading: true,
+        isError: false,
+        error: null,
+        isSuccess: false,
+        inputError: null,
+      };
+    case 'INPUT_FAILURE':
+      return {
+        ...initialState,
+        isLoading: false,
+        isError: true,
+        error: null,
+        isSuccess: false,
+        inputError: action.inputError,
+      };
+    default:
+      throw Error('Unknown action type');
+  }
+};
+
+function CloseConfirmAlert() {
+  const [state, dispatch] = useReducer(dataFetchReducer, initialState);
+
+  const classes = useStyles();
+
+  const [title, setTitle] = useState('');
+  const [description, setDescription] = useState('');
+  const [checked, setChecked] = React.useState(true);
+
+  const handleChange = (event: React.ChangeEvent<HTMLInputElement>) => {
+    setChecked(event.target.checked);
+  };
+
+  function closeAlert() {
+    if (!isTextFieldValuesValid(title)) {
+      dispatch({
+        type: 'INPUT_FAILURE',
+        miniAppError: null,
+        inputError: 'Title cannot be empty',
+      });
+      return;
+    }
+    if (!isTextFieldValuesValid(description)) {
+      dispatch({
+        type: 'INPUT_FAILURE',
+        miniAppError: null,
+        inputError: 'Description cannot be empty',
```

Comment:
I am not sure if we should force to give non-empty inputs 🤔 
I am planning to add default empty string values in Android side.

Reply:
@khairul-alam-licon . This is just a normal validation that we do for our Sample app. I believe Android/iOS platforms can display empty strings for title & description.